codalab/lib/formatting.py

```diff
@@ -22,6 +22,8 @@ def contents_str(input_string):
     if input_string is None:
         return ''
 
+    if isinstance(input_string, unicode):
+        return input_string
     try:
         input_string.decode('utf-8')
```

Comment:
This does get used still in some places to mask strings that contain binary (non-unicode) content, stored in plain strings.